lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -64,12 +64,22 @@
          CA server API."
       - "For safety reasons the default is set to the Let's Encrypt staging server.
          This will create technically correct, but untrusted certificates."
-    default: https://acme-staging.api.letsencrypt.org/directory
+    default: https://acme-staging-v02.api.letsencrypt.org/directory
+  acme_version:
+    description:
+      - "The ACME version of the endpoint."
+      - "Must be 1 for the classic Let's Encrypt ACME endpoint, or 2 for the
+         new ACME v2 endpoint."
+    default: 2
+    choices: [1, 2]
+    version_added: "2.5"
   agreement:
     description:
-      - "URI to a terms of service document you agree to when using the
-         ACME service at C(acme_directory)."
-      - Default is latest gathered from C(acme_directory) URL.
+      - "Boolean indicating whether you agree to the terms of service document."
+      - "ACME servers can require this to be true."
+      - "Note: this setting was changed in version 2.5 to be a boolean, and no
+         longer is the URL of the terms of service document."
+    default: false
```

Comment:
Changing an argument will break users playbooks, which won't be accepted.

Please create a new argument for this e.g. `term_agreed`.

Reply:
Fixed that. I also changed the defaults for `acme_version` and `acme_directory` to `1` resp. the ACME v1 staging endpoint.

Reply:
NB: the current devel letsencrypt module already breaks existing playbooks due to #22074 (default of `fullchain` is `true`, even though documentation says its `false`).